### PR TITLE
[zh] Sync #14387 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14387 into Chinese

## Reviewers

@windsonsea please help to review, I think some sentences are not good enough.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

**Comment by windsonsea** [comment]
Need to fix the lint errors:
```
./zh/boilerplates/helm-preamble.md:7: MD009 Trailing spaces
```

File: content/zh/boilerplates/helm-preamble.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ---
 ---
-本指南中使用的 Helm Chart 与通过 [Istioctl](/zh/docs/setup/install/istioctl/)
-或 [Operator](/zh/docs/setup/install/operator/) 安装 Istio 时使用的底层 Chart 相同。
+通过使用本指南中 `base` 和 `istiod` Helm Chart 与通过
+[Istioctl](/zh/docs/setup/install/istioctl/) 或
+[Operator](/zh/docs/setup/install/operator/) 安装 Istio 相同。
```
Comment (windsonsea) [inline content/zh/boilerplates/helm-preamble.md:5]:
```suggestion
本指南中使用的 `base` 和 `istiod` Helm Chart 与通过
[Istioctl](/zh/docs/setup/install/istioctl/) 或
[Operator](/zh/docs/setup/install/operator/) 安装 Istio 时使用的 Chart 相同。
```

File: content/zh/boilerplates/helm-preamble.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+但是，通过 Istioctl 和 Operator 进行的安装时使用了与本指南描述中
+[Chart]({{< github_tree >}}/manifests/charts/gateway)
```
Comment (windsonsea) [inline content/zh/boilerplates/helm-preamble.md:7]:
```suggestion
但是，通过 Istioctl 和 Operator 进行安装时使用了与本指南所述
[Chart]({{< github_tree >}}/manifests/charts/gateway)
```

**Review by windsonsea — COMMENTED** [review]
Some text suggestions

File: content/zh/boilerplates/helm-preamble.md (hunk 2 above)
Comment (wilsonwu) [reply content/zh/boilerplates/helm-preamble.md]: applied

File: content/zh/boilerplates/helm-preamble.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/boilerplates/helm-preamble.md]: applied

**Review by windsonsea — APPROVED** [review]
lgtm

**Pull request merged**